README.rst

```diff
@@ -176,7 +176,8 @@ command::
 
 To run the application, use the ``paste`` script with the ``serve`` command.
 (The paste script and serve command come from ``PasteScript`` and
-``PasteDeploy``, respectively.)
+``PasteDeploy``, respectively, which will need to be installed: ``pip install pastescript``
+``pip install pastedeploy``.)
```

Comment:
It's showing the age of this documentation, but this is no longer true. The real command to use is Pyramid's `pserve`, which deep down contains forked bits of PasteScript and PasteDeploy. So the statement above isn't altogether false, but clarification is definitely needed.